integrationtests/volume_test.go

```diff
@@ -212,11 +246,6 @@ func negativeDiskTests(t *testing.T) {
 		t.Fatalf("Client new error: %v", err)
 	}
 	defer client.Close()
-
-	// Empty disk Id test
-	runNegativeListVolumeRequest(t, client, "")
-	// Negative disk id test
-	runNegativeListVolumeRequest(t, client, "-1")
```

Comment:
why remove this test?

Reply:
the API no longer receives strings or negative numbers because it's an unsigned int field now :), I think pre v1beta3 tests could be added again once I work on the e2e tests